README.md

```diff
@@ -4,6 +4,6 @@ A work-in-progress, healthy alternative to the Stack Exchange Network.
 
 This project was created to coordinate and keep track of current efforts by a group of (mostly unsatisfied) users of Stack Overflow and the Stack Exchange Network to develop an alternative to these platforms.
```

Comment:
While we're in there anyway, let's replace this with: This project was created to coordinate current efforts to develop a new community platform  (or something like that).